edumanage/forms.py

```diff
@@ -97,8 +100,10 @@ def clean_acct_port(self):
 
     def clean_rad_pkt_type(self):
         rad_pkt_type = self.cleaned_data['rad_pkt_type']
-        institution = self.cleaned_data['instid']
-        if institution.ertype in [1,3]:
+	if not 'ertype' in self.cleaned_data:
+                raise forms.ValidationError(_('The Type field is required to validate this field.'))
+        ertype = self.cleaned_data['ertype']
+        if ertype in [1,3]:
```

Comment:
This field is actually required even if the institution/server is SP only, because NRO servers may need to know whether they should handle accounting packets. We want to deprecate accounting and thus get rid of RADTYPES in the future, but this is how things stand for now. So I believe you should remove any 'ertype' checking in this function.